crates/teepot/tests/sgx_quote_fmspc.rs

```diff
@@ -0,0 +1,1394 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2024-2025 Matter Labs
+
+mod fmspc {
+    use anyhow::Result;
+    use teepot::quote::{Fmspc, Quote};
+
+    fn check_quote(raw_quote: &[u8], expected_result: &Fmspc) -> Result<()> {
+        let quote = Quote::parse(raw_quote).unwrap();
+        let fmspc = quote.fmspc().unwrap();
+        assert_eq!(&fmspc, expected_result);
+        Ok(())
+    }
+
+    #[test]
+    // alternative quote verification cannot cope with old collateral data format
+    fn sw_hardening() {
+        let quote = [
```

Comment:
Surely it's better to put these as a binary file somewhere and `include_bytes!` them?

Reply:
I'll file an issue.

Reply:
https://github.com/matter-labs/teepot/issues/308